frontend/util.lua

```diff
@@ -984,6 +984,7 @@ If an optional path is provided, @{util.getFilesystemType}() will be used to det
 ---- @treturn string safe filename
 function util.getSafeFilename(str, path, limit, limit_ext)
     local filename, suffix = util.splitFileNameSuffix(str)
+    filename = filename:gsub("\r?\n", " "):gsub("\t", " ")
```

Comment:
Would it make more sense conceptually around line 1010 (just before the length limit)? Fine by me either way.

Reply:
Agree